native/dgebak.go

```diff
@@ -0,0 +1,79 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package native
+
+import (
+	"github.com/gonum/blas"
+	"github.com/gonum/blas/blas64"
+	"github.com/gonum/lapack"
+)
+
+// Dgebak transforms an n×m matrix V as
+//  V = P D V,        if side == blas.Right,
```

Comment:
Can you use "updates"? I was thinking "transform" was some kind of eigensolution (finds the vector V such that...)